### PR TITLE
✨ respect Sentry CaptureConsole level settings

### PR DESCRIPTION
**Pull request by jeysal**
* For https://github.com/playt-net/clashparadise/issues/3962
* Depends on https://github.com/playt-net/clashparadise/pull/4061

### CODE REVIEW

**Comment by jeysal** [comment]
Tested by linking into Rolling Rick, adding a warning to Rolling Rick, and messing with the levels returned by the Clash Paradise API

File: src/types.mts [hunk 1]
```diff
@@ -324,9 +353,7 @@ export interface operations {
       /** Successful response */
       200: {
         content: {
-          'application/json': {
-            replayId: string;
-          };
+          'application/json': unknown;
```
Comment (jeysal) [inline src/types.mts:356]: strange API change coming out of the openapi typescript generator o.O

**Comment by jeysal** [comment]
@playt-net/developer review pls

File: src/types.mts (hunk 1 above)
Comment (jeysal) [reply src/types.mts:356]: @boredland could this come from the replay entity->field refactor?

**Review by franzosterkamp — APPROVED** [review]

**Pull request merged**